modules/apps/data-engine/data-engine-rest-impl/src/test/java/com/liferay/data/engine/rest/internal/strategy/DefaultMapToDDMFormValuesConverterStrategyTest.java

```diff
@@ -57,6 +59,41 @@ public void setUp() {
 		_setUpLanguageUtil();
 	}
 
+	@Test
+	public void testCreateDDMFormFieldValueAsHashMap() {
+		DDMFormField ddmFormField = _createDDMFormField(
+			"field1", "document_library", true);
+
+		Map<String, Object> innerValues =
```

Comment:
Could you rename this variable to "values" ?

Reply:
The customer is the boss! :+1: